test/ruby/test_gc_compact.rb

```diff
@@ -210,6 +210,7 @@ def obj.tracee
   end
 
   def test_moving_arrays_down_size_pools
+    omit if GC::INTERNAL_CONSTANTS[:SIZE_POOL_COUNT] == 1
     omit if !GC.using_rvargc?
```

Comment:
I think you can remove this line since `GC::INTERNAL_CONSTANTS[:SIZE_POOL_COUNT] == 1` when `GC.using_rvargc? == false`

Reply:
Yup, the goal here was to make it easier when we remove the non rvargc code path. We'll still need the `omit if GC::INTERNAL_CONSTANTS[:SIZE_POOL_COUNT] == 1` to stay. Happy to remove it though if you think that's better, and then when we remove the non rvargc code path, we'll need to add it back.

Reply:
I meant removing the `GC.using_rvargc? == false` part (maybe we can get rid of the method `GC.using_rvargc?` altogether), not the `GC::INTERNAL_CONSTANTS[:SIZE_POOL_COUNT] == 1` part.